strongbox-data-service/src/main/java/org/carlspring/strongbox/config/DataServiceConfig.java

```diff
@@ -107,15 +78,16 @@ public EntityManagerFactory entityManagerFactory()
 
         LocalContainerEntityManagerFactoryBean result = new LocalContainerEntityManagerFactoryBean();
         result.setJpaPropertyMap(jpaProperties);
-        result.afterPropertiesSet();
+        result.setDataSource(dataSource);
+        result.setPersistenceUnitName("strongbox-PU");
+        result.setPackagesToScan("org.carlspring.strongbox");
+        result.setPersistenceProvider(new OJPAObjectDatabaseTxPersistenceProvider());
 
-        return result.getObject();
-    }
+        // @sbespalov, needed ?
+        // ServiceLoader.load(OSQLFunctionFactory.class);
```

Comment:
if it works without it then I think it don't needed any more :)

Reply:
Let's check ;)